python/uv/_build_backend.py

```diff
@@ -18,17 +18,17 @@
 
 TYPE_CHECKING = False
 if TYPE_CHECKING:
-    from typing import Any  # noqa:I001
+    from typing import Any, Mapping  # noqa:I001
```

Comment:
`typing.Mapping` is deprecated since python 3.9, so it's recommended to import it from `collections.abc`: https://docs.python.org/3/library/typing.html#typing.Mapping

And it apparently already works on Python 3.8:
https://basedpyright.com/?pythonVersion=3.8&typeCheckingMode=all&code=GYJw9gtgBAxmA28CmMAuBLMA7AzgOgEMAjGKdCABzBFSgFkCKL0sBzAWACguCAuexszYBtHKhAAaKGJABdKAF4oAbwC%2BXIA